spec/tasks/rake_spec.rb

```diff
@@ -12,21 +12,21 @@
     end
```

Comment:
Honestly, I don't know how to test stuff that's reading from `$stdin`.  This code is copied from https://github.com/projectblacklight/spotlight/blob/master/lib/tasks/spotlight_tasks.rake#L2 and it wasn't tested their either.